### ISSUE TITLE
1.9.2-RC2: Macos m1 Failed to load the librdkafka native library.

### ISSUE THREAD

**Issue by niemyjski** [confluentinc/confluent-kafka-dotnet#1865]
Description
===========

Using https://www.nuget.org/packages/Confluent.Kafka/1.9.2-RC2, gives me

```
System.DllNotFoundException
Failed to load the librdkafka native library.
   at Confluent.Kafka.Impl.Librdkafka.TrySetDelegates(List`1 nativeMethodCandidateTypes)
   at Confluent.Kafka.Impl.Librdkafka.LoadOSXDelegates(String userSpecifiedPath)
   at Confluent.Kafka.Impl.Librdkafka.Initialize(String userSpecifiedPath)
   at Confluent.Kafka.Producer`2..ctor(ProducerBuilder`2 builder)
   at Confluent.Kafka.ProducerBuilder`2.Build()
```

on Macbook M1 Pro.

How to reproduce
================

Checklist
=========

Please provide the following information:
 - [ ] A complete (i.e. we can run it), minimal program demonstrating the problem. No need to supply a project file.
 - [x] Confluent.Kafka nuget version. 1.9.2-rc
 - [x] Apache Kafka version. latest
 - [ ] Client configuration.
 - [ ] Operating system.
 - [ ] Provide logs (with "debug" : "..." as necessary in configuration).
 - [ ] Provide broker log excerpts.
 - [x] Critical issue.

**Comment by edenhill** [comment]
RC2 lacks an M1 build, will be available in a later RC

**Comment by mhowlett** [comment]
resolved in 1.9.2
